lambdatimes/src/Authentication/Authenticate.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
```

Comment:
i think you would want the authenticate component to wrap your app, and have this component have the state of loggedIn, where then if the user is logged in or not, you can conditionally render either a login modal/component or the app. in the login component you can then set local storage once you hit submit.